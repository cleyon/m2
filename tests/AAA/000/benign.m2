@# Called out in the man page as not triggering any changes
100 dollars @ 5% annual interest
