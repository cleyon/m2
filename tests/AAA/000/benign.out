100 dollars @ 5% annual interest
